src/browser/BrowserService.cpp

```diff
@@ -704,23 +765,38 @@ BrowserService::checkAccess(const Entry* entry,
     return Unknown;
 }
 
+bool BrowserService::insideRecycleBin(Group* group)
```

Comment:
This should be a member of the group class: ```Group::isRecycled()```

Reply:
This is now done. I also added tests for this to the TestGroups.